src/main/scala/com/kifi/franz/SQSQueue.scala

```diff
@@ -8,16 +8,7 @@ import scala.collection.JavaConverters._
 import scala.language.implicitConversions
 
 import com.amazonaws.services.sqs.AmazonSQSAsync
-import com.amazonaws.services.sqs.model.{
-  SendMessageRequest,
-  GetQueueUrlRequest,
-  ReceiveMessageRequest,
-  DeleteMessageRequest,
-  SendMessageResult,
-  ReceiveMessageResult,
-  CreateQueueRequest,
-  MessageAttributeValue
-}
+import com.amazonaws.services.sqs.model._
```

Comment:
Yeah that was probably my IDE being intelligent or the scalastyle plugin.
I will fix.
Amit

On Jun 2, 2015, at 10:42, Stephen Kemmerling <notifications@github.com> wrote:

> In src/main/scala/com/kifi/franz/SQSQueue.scala:
> 
> > @@ -8,16 +8,7 @@ import scala.collection.JavaConverters._
> >  import scala.language.implicitConversions
> >  
> >  import com.amazonaws.services.sqs.AmazonSQSAsync
> > -import com.amazonaws.services.sqs.model.{
> > -  SendMessageRequest,
> > -  GetQueueUrlRequest,
> > -  ReceiveMessageRequest,
> > -  DeleteMessageRequest,
> > -  SendMessageResult,
> > -  ReceiveMessageResult,
> > -  CreateQueueRequest,
> > -  MessageAttributeValue
> > -}
> > +import com.amazonaws.services.sqs.model._
> I'm not a fan of _ imports. Makes the code harder to reason about.
> 
> —
> Reply to this email directly or view it on GitHub.
>